study-builder/tenants/register-user-in-pepper.js

```diff
@@ -136,6 +136,20 @@ function (user, context, callback) {
             console.log('No Registration Mode passed in request');
         }
 
+        if (pepper_params.mode && pepper_params.mode === "login" &&
```

Comment:
During login check user_guid or temp_user_guid to check that user was registered before trying to login.
Block any further progress and return back if user was NOT registered